lib/ood_core/job/adapters/slurm.rb

```diff
@@ -454,6 +473,12 @@ def submit(script, after: [], afterok: [], afternotok: [], afterany: [])
           raise JobAdapterError, e.message
         end
 
+        # Retrieve info about active and total cpus, gpus, and nodes
+        # @return [Hash] information about cluster usage
+        def cluster_info()
+          @slurm.get_cluster_info()
+        end
```

Comment:
The ruby-ish way to write this is without parentheses. 

```suggestion
        def cluster_info
          @slurm.get_cluster_info
        end
```